packs/bit-bucket/requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.6.0
```

Comment:
@Kami, Yep I looked at it, but then went for using REST API, because I though the library might be limited in functionality and if we further need to extend the pack then we can extend it just by adding a call to API endpoint rather than waiting for the python package to be upgraded to the latest release for incorporating the latest changes.